Assets/StreamingAssets/Configs/carrier_strike_group_example.json

```diff
@@ -0,0 +1,131 @@
+      "velocity": { "x": 0, "y": 0, "z": -12 },
+      "max_interceptors": 8,
+      "interceptor_types": ["hydra70.json", "sfrj.json"],
+      "standard_deviation": {
+        "position": { "x": 100, "y": 0, "z": 150 },
+        "velocity": { "x": 1, "y": 0, "z": 1 }
+      }
+    },
+    {
+      "id": "CG-47-Ticonderoga",
+      "type": "Ship",
+      "initial_position": { "x": 3000, "y": 0, "z": 6000 },
+      "velocity": { "x": 0, "y": 0, "z": -12 },
+      "max_interceptors": 12,
+      "interceptor_types": ["hydra70.json", "sfrj.json"],
+      "standard_deviation": {
+        "position": { "x": 75, "y": 0, "z": 125 },
+        "velocity": { "x": 1, "y": 0, "z": 1.5 }
+      }
+    },
+    {
+      "id": "Aegis-Ashore-Guam",
+      "type": "ShoreBattery",
+      "initial_position": { "x": 15000, "y": 100, "z": 0 },
+      "velocity": { "x": 0, "y": 0, "z": 0 },
+      "max_interceptors": 48,
+      "interceptor_types": ["hydra70.json", "sfrj.json", "patriot.json"],
+      "standard_deviation": {
+        "position": { "x": 200, "y": 0, "z": 200 },
+        "velocity": { "x": 0, "y": 0, "z": 0 }
+      }
+    }
+  ],
+  "origin_assignment_strategy": "CLOSEST",
+  "interceptor_swarm_configs": [
+    {
+      "dynamic_agent_config": {
+        "agent_model": "hydra70.json",
+        "dynamic_config": {
+          "sensor_config": {
+            "type": "IDEAL",
+            "frequency": 100
+          },
+          "flight_config": {
+            "augmentedPnEnabled": false
+          }
+        },
+        "submunitions_config": {
+          "num_submunitions": 7,
+          "dynamic_agent_config": {
+            "agent_model": "micromissile.json",
+            "dynamic_config": {
+              "sensor_config": {
+                "type": "IDEAL",
+                "frequency": 100
+              },
+              "flight_config": {
+                "augmentedPnEnabled": false
+              }
+            }
+          }
+        }
+      }
+    }
+  ],
+  "threat_swarm_configs": [
+    {
+      "num_agents": 22,
+      "dynamic_agent_config": {
+        "agent_model": "brahmos.json",
+        "attack_behavior": "brahmos_direct_attack.json",
+        "initial_state": {
+          "position": { "x": 0, "y": 50, "z": 40000 },
+          "velocity": { "x": 0, "y": 0, "z": -2000 }
+        },
+        "standard_deviation": {
+          "position": { "x": 20000, "y": 10, "z": 10000 },
+          "velocity": { "x": 0, "y": 0, "z": 100 }
+        },
+        "dynamic_config": {
+          "launch_config": { "launch_time": 0 },
+          "sensor_config": {
+            "type": "IDEAL",
+            "frequency": 10
+          },
+          "flight_config": {
+            "augmentedPnEnabled": false,
+            "evasionEnabled": false
+          }
+        },
+        "plotting_config": {
+          "color": "RED",
+          "linestyle": "SOLID",
+          "marker": "TRIANGLE_DOWN"
+        }
+      }
+    }
+  ]
+}
```

Comment:
add newline

also would remove the `_example` suffix in the file and add some description about the engagement like `carrier_strike_grouop_brahmos.json`